public/js/p3/widget/app/CoreGenomeMLST.js

```
@@ -0,0 +1,300 @@
+        return this.inherited(arguments);
+      },
+
+      validateGenomeGroup: function () {
+        // this.submitButton.set('disabled', true);
+        var path = this.select_genomegroup.searchBox.item.path;
+        var genomes_valid = true;
+        when(WorkspaceManager.getObject(path), lang.hitch(this, function (res) {
+          if (typeof res.data == 'string') {
+            res.data = JSON.parse(res.data);
+          }
+          if (res && res.data && res.data.id_list && res.data.id_list.genome_id) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)